src/openfga.ne

```diff
@@ -0,0 +1,98 @@
+@preprocessor typescript
+
+types           -> (_newline):* (type (relations (define):+):*):* {%
+    data => data[1].map((datum: any) => {
+		    const relations = datum[1][0] ? datum[1][0][1].flat() : [];
+		    return { ...datum[0], relations }
+		})
+%}
+
+type            -> _multiline_comment _type _naming (_newline):+ {%
+    data => ({ comment: data[0], type: data[2] })
+%}
+relations       -> _multiline_comment _relations {%
+    data => data[1]
+%}
+define          -> _multiline_comment define_initial _as (define_base | define_or | define_and | define_but_not) (_newline):* {%
```

Comment:
how does the multiline comment work? 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/5374887/191821201-2a085a29-29b4-4783-a931-fb254f01dcba.png">

Reply:
This should be fixed now - tests for comments will come later, after we fix the checkDSL